slasher/server.go

```diff
@@ -0,0 +1,65 @@
+package slasher
+
+import (
+	"context"
+	"github.com/pkg/errors"
+
+	"github.com/gogo/protobuf/proto"
+	types "github.com/gogo/protobuf/types"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
+	ethpb "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
+	"github.com/prysmaticlabs/prysm/slasher/db"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// Server defines a server implementation of the gRPC Slasher service,
+// providing RPC endpoints for retrieving slashing proofs for malicious validators.
+type Server struct {
+	slasherDb *db.Store
+	ctx       context.Context
```

Comment:
any reason's for having context as a field here ?

Reply:
what about closing the service. wouldn't i need it?

Reply:
it would only be needed if you are listening for ctx.Done somewhere, but it doesnt look like
its implemented in this service